### ISSUE TITLE
Accessibility: 

### ISSUE THREAD

**Issue by Abolade017** [jupyterhub/outreachy#73]
### What page is this for?

https://jupyter.org/security#security-documentation

### WAVE accessibility report

I evaluated the web accessibility of this page with WAVE tool. The issues and it's fixing are the following: 

![security](https://user-images.githubusercontent.com/70269933/196198240-2ba33933-41bd-426e-9394-5add483b9ac4.JPG)

 ##**Evaluation**

- **Alternative text (1) :**:  A non-empty `alt` attribute is present on an image. This issue can be fixed by ensuring that the alternative text conveys the content and function of the image accurately and succinctly. The alt attribute should be equivalent, accurate, and succinct.
- **Linked image with alternative text (1) :** An image element has non-empty alternative text, is within a link, and no other text (or images with alternative text) is present within the link. This issue can be solved by ensuring that the alternative text conveys the content and function of the image accurately and succinctly. The alt attribute should be equivalent, accurate, and succinct.
- **Language (1) :** A document or an element has a valid `lang` attribute value. this issue can be fixed by ensuring the language is properly identified for the page (e.g., `<html lang="en">`). If content within the page is in a language different than the page's language, identify it using a valid `lang` attribute value (e.g., `<p lang="fr">`).
- **Heading level (3) :** `<h1>` and `<h2>` element is present. This issue can be fixed by ensuring that the text in question is truly a heading and that it is structured correctly in the page outline.
- **Unordered list (5) :** The `<ul>` element is present. this is issue can be fixed by ensuring that an unordered (bulleted) list is appropriate for the context. If list items are sequential or numbered, an ordered list (`<ol>`) is likely more appropriate. 
- **Navigation (1) :** A `<nav>` element or `role="navigation"` is present. this issue can be fixed by ensuring that the element defines page navigation. Multiple navigation elements on one page can be differentiated with `ARIA` labels.
- **Footer (1) :** A `<footer>` element or `role="contentinfo"` is present. This issue can be fixed by ensuring the element surrounds and defines page or page section footer content.
- **ARIA (7) :** An `aria-label` or `aria-labelledby` attribute is present. This issue can be fixed by ensuring that the `aria-label` value or element referenced by `aria-labelledby` provides a descriptive, accurate label. When possible, use standard HTML <label> or other markup to make the association.

### The HTML element to be changed

```html
1. These landmark used <nav> and <section> are to be changed.
2. header should be remove as <h1>Security</h1>
  <p>Project Jupyter is committed to reducing risk in using, deploying, operating, or developing Jupyter software.</p>
  <script src="https://cdn.jsdelivr.net/npm/anchor-js/anchor.min.js"></script>
  <script>
    document.addEventListener('DOMContentLoaded', function(event) {
      anchors.add();
    });
  </script> parent element.
2. The <article> element after the <header> is to be changed. 
3. <a href="https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/about-alerts-for-vulnerable-dependencies">GitHub</a> and <a href="https://github.com/jupyter/" target="_blank" rel="noopener noreferrer">GitHub</a>. One of the two accessibility name is to be changed.  
4. missing "lang=fr" attribute in <a href="https://voila.readthedocs.io/" target="_blank" rel="noopener noreferrer">Voilà</a>
5. The <a class="anchorjs-link " aria-label="Anchor" data-anchorjs-icon="" href="#community-resources" style="font: 1em / 1 anchorjs-icons; padding-left: 0.375em;"></a>, the aria-label text are the same through out the page which should not be so because it doesn't describe the functionality of each heading
```

### Your proposed HTML (or CSS), after change

```html
1. I will put the <nav> and <h1>Security</h1>
  <p>Project Jupyter is committed to reducing risk in using, deploying, operating, or developing Jupyter software.</p>
  <script src="https://cdn.jsdelivr.net/npm/anchor-js/anchor.min.js"></script>
  <script>
    document.addEventListener('DOMContentLoaded', function(event) {
      anchors.add();
    });
  </script> inside the <header> as their parent element. 
2. I change the <article> element after the <header> into <main> and use <article> element as a descendant element of <main>. 
3. I will change the accessibility name of <a href="https://github.com/jupyter/" target="_blank" rel="noopener noreferrer">GitHub</a> to "jupyter GitHub".
4. I will add "lang=fr" attribute to <a href="https://voila.readthedocs.io/" target="_blank" rel="noopener noreferrer">Voilà</a>
5. I will change the aria-label value to an appropriate text for each <a class="anchorjs-link " aria-label="Anchor" data-anchorjs-icon="" href="#community-resources" style="font: 1em / 1 anchorjs-icons; padding-left: 0.375em;"></a> on the page.
```

**Comment by minrk** [comment]
Thank you for these suggestions!
